packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -80,21 +81,19 @@ _ref:
             The config object has the following properties:
 
             - `basePath: string`: Set the base path to serve the Lowdefy application from. This will route all pages under `https://example.com/<base-path>/<page-id>` instead of the default `https://example.com/<page-id>`. The basePath value must start with "/".
-            - `homePageId: string`: The pageId of the page that should be loaded when a user loads the app without a pageId in the url route. This is the page that is loaded when you navigate to `yourdomain.com`.
+            - `homePageId: string`: The id of the page that will load when a visitor navigates to the home route, for example, `yourdomain.com`.
```

Comment:
I think the "for example, `yourdomain.com`." part might be a bit misleading, which is why the original explanation is so verbose. This is specifically if the user navigates to `example.com` or `example.com/`, not `example.com/page-id`.

Reply:
Change made in 386463869c78c494c05636e1e435419c96917b02 
Tried to get a bit more of a balance between the two versions.